01-shapefile-attributes.Rmd

```diff
@@ -51,7 +52,7 @@ More on Packages in R - Adapted from Software Carpentry.</a>
 
 ##Data to Download
 
-Download the shapefiles neede to complete this lesson:
+Download the shapefiles needes to complete this lesson:
```

Comment:
shapefiles needed to complete this lesson. @mjones01 can you tell me if this comment directs you right to the line where this issue is so you can fix it?